src/pages/Home/StartPrefectServer-Section.vue

```diff
@@ -114,14 +124,32 @@ export default {
           @keyup="_handleKeyup"
         >
           <template v-slot:append>
-            <v-fade-transition mode="out-in">
-              <v-icon v-if="success" key="success" color="green">
-                check
-              </v-icon>
-              <v-icon v-else-if="error" key="error" color="error">
-                error
-              </v-icon>
-            </v-fade-transition>
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  color="blue-grey lighten-1"
+                  icon
+                  small
+                  @click="_resetUrl"
+                  v-on="on"
+                >
+                  <v-fade-transition mode="out-in">
```

Comment:
I find the mix of indicator and action button a bit confusing here.  Could we keep the indicator as is and add a reset button next to the connect button?  (Or something similar.)